doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -737,6 +737,14 @@ Other libraries
 ---------------
 
 * Added a compression/decompression library with support for the LZMA decompression.
+* :ref:`lib_date_time` library:
+
+  * Fixed a bug that caused date-time updates to not be rescheduled under certain circumstances.
+
+  * Added:
+
+    * A retry feature that reattempts failed date-time updates up to a certain number of consecutive times.
+    * The Kconfig options :kconfig:option:`CONFIG_DATE_TIME_RETRY_COUNT`, to control whether and how many consecutive date-time update retries may be performed, and :kconfig:option:`CONFIG_DATE_TIME_RETRY_INTERVAL_SECONDS`, to control how quickly date-time update retries occur.
```

Comment:
```suggestion
    * The Kconfig options :kconfig:option:`CONFIG_DATE_TIME_RETRY_COUNT` to control whether and how many consecutive date-time update retries may be performed, and :kconfig:option:`CONFIG_DATE_TIME_RETRY_INTERVAL_SECONDS` to control how quickly date-time update retries occur.
```

Reply:
These commas are grammatically necessary, are they not? They are equivalent to the construction "The options A (for xyz), and B (for qed)"

But sure, I'll remove them